learn/avalanche-bridge-faq.md

```diff
@@ -46,6 +46,14 @@ Yes, you can click on the “Speed Up” button on Metamask. “Speeding up” a
 
 When transferring from Avalanche to Ethereum, Metamask shows that 0 tokens are to be transferred, not the actual number of tokens. This is a known problem with Metamask.
 
+#### What are the bridge contract adresses? 
+
+Bridge Addresses:
+    - ETH: `0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0` https://etherscan.io/address/0xe78388b4ce79068e89bf8aa7f218ef6b9ab0e9d0
+    - AVAX: `0x50Ff3B278fCC70ec7A9465063d68029AB460eA04` https://cchain.explorer.avax.network/address/0x50Ff3B278fCC70ec7A9465063d68029AB460eA04
```

Comment:
@salwao2 it is weird that github made this mess. Can you remove the lines 52-53?